pyproject.toml

```diff
@@ -100,77 +100,55 @@ line-length = 88
 
 [tool.ruff]
 line-length = 88
+target-version = "py39"
+extend-exclude = ["doc/*", "benchmarks/*", "versioneer.py", "geopandas/_version.py"]
+
+[tool.ruff.lint]
 select = [
     # pyflakes
     "F",
     # pycodestyle
     "E",
     "W",
-    # flake8-2020
-    "YTT",
+    # pyupgrade
+    # "UP",
     # flake8-bugbear
     "B",
-    # flake8-quotes
-    "Q",
```

Comment:
This essentially checks for double vs single quotes, but AFAIK that is already enforced by the formatter